adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -270,7 +270,7 @@ class ADAMContextSuite extends ADAMFunSuite {
   sparkTest("can read a small .vcf file") {
     val path = resourcePath("small.vcf")
 
-    val vcs = sc.loadGenotypes(path).toVariantContext.collect.sortBy(_.position)
+    val vcs = sc.loadGenotypes(path).toVariantContextRDD.rdd.collect.sortBy(_.position)
```

Comment:
Sorry, what I mean was that sometimes the ```genomicRDDToRDD``` implicit works for me and sometimes it doesn't.  I assume because you are using ```.rdd``` here that they are necessary.  The scala compiler and I are still getting to know each other.